docs/languages/scala.md

```diff
@@ -35,7 +35,12 @@ M.config = function()
     excludedPackages = {},
   }
   metals_config.init_options.statusBarProvider = false
-  require("metals").initialize_or_attach { metals_config }
+  require("metals").initialize_or_attach(metals_config)
+  vim.api.nvim_create_autocmd({ "BufEnter", "BufWinEnter" }, {
```

Comment:
Please specify an augroup!

Also, how does this compare to using their [recommended way](https://github.com/scalameta/nvim-metals/discussions/39) with a `FileType` event?

```lua
api.nvim_create_autocmd("FileType", {
  -- NOTE: You may or may not want java included here. You will need it if you
  -- want basic Java support but it may also conflict if you are using
  -- something like nvim-jdtls which also works on a java filetype autocmd.
  pattern = { "scala", "sbt", "java" },
  callback = function()
    require("metals").initialize_or_attach(metals_config)
  end,
  group = nvim_metals_group,
})
```

Reply:
I don't have any strong opinion about `BufEnter`. If it's recommended by `nvim-metals` I'd happily change into this. Basically what I'd done was continue whatever has been written in previous documentation and fix whatever was necessary.
Not really familiar with `autogroup`, any link I could read?

Reply:
Addressed.